libs/logic-apps-shared/src/designer-client-services/lib/operationmanifest.ts

```diff
@@ -36,6 +36,13 @@ export interface IOperationManifestService {
    * @return {Promise<any>}
    */
   getOperationManifest(connectorId: string, operationId: string): Promise<OperationManifest>;
+
+  /**
+   * Gets the transformed output property body items with aliases
+   * @arg {OpenAPIV2.SchemaObject | undefined} schemaItem - The output property body items.
+   * @return {OpenAPIV2.SchemaObject | undefined}
+   */
+  getUpdatedAliasInOutputItemProperties?(schemaItem: OpenAPIV2.SchemaObject | undefined): OpenAPIV2.SchemaObject | undefined;
```

Comment:
PLease name this as - getUpdatedOutputsForSplitOn

This way your method can take full control ovber the manifest schema which may or maynot include for item properties.

Reply:
Not sure I follow here - is it just a matter of function rename or increasing the scope of the service function? I would rather not duplicate code in the host if not necessary unless LA is not using most of the split on logic. To my understanding the aliasing logic was the bit LA wasn't using.

Reply:
Renamed the function